neurom/check/neuron_checks.py

```diff
@@ -223,6 +223,19 @@ def has_no_jumps(neuron, max_distance=30.0, axis='z'):
     return CheckResult(len(bad_ids) == 0, bad_ids)
 
 
+def has_no_root_node_jumps(neuron, radius_multiplier=2):
```

Comment:
Not sure about the name of the function

Reply:
Not sure about the name `radius_multiplier`

Reply:
I can't think of better names